governance.md

```diff
@@ -1,149 +1,285 @@
-  for accepting the contribution of a new component or other major function
-- [**kubernetes-maintainers**](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers):
-  approver for some part of the codebase for at least 3 months; on project for at least 1 year;
-  at least 50 merged and/or assigned PRs, including at least 20 as the primary reviewer;
-  write access to repo (assign issues/PRs, add/remove labels and milestones, edit issues and PRs, edit wiki,
-  create/delete labels and milestones); technically can lgtm any PR and cause it
-  to be merged by the submit queue, but expected to respect OWNERS files; expected to review PRs, fix bugs, maintain and
-  improve health and quality of the project, provide user support, mentor and guide approvers,
-  reviewers, and other contributors; must apply to `kubernetes-maintainers@googlegroups.com`, with a
-  [Champion](https://github.com/kubernetes/community/blob/master/incubator.md#faq) from the existing
-  kubernetes-maintainers members and a Sponsor from Project Approvers, with a summary
-  of contributions to the project, current project responsibilities, and links to merged and assigned PRs;
-  at least 3 of the maintainers must approve the application, with no objections; the application
-  expires after 2 weeks if not enough approvals are granted
-- **Project Approvers**: approver in [top-level OWNERS file in kubernetes repo](https://github.com/kubernetes/kubernetes/blob/master/OWNERS);
-  de-facto project decision makers; technically can 
-  approve virtually any PRs; can sponsor incubator repos; can sponsor maintainers;
-  maintainer in good standing for at least 1 year; strong technical vision;
-  committed to project's mission and culture; nomination/application process TBD
-- [**API Approver**](https://github.com/orgs/kubernetes/teams/api-approvers):
-  lead designers of the project, who are familiar with the 
-  design, requirements, mechanics, conventions, style, scope, gotchas, etc. 
-  of the API; most beta/GA API changes are vetted by the API approvers
-- [**API Reviewer**](https://github.com/orgs/kubernetes/teams/api-reviewers):
-  contributors familiar with design, requirements, mechanics, conventions, style,
-  scope, gotchas, etc. of the API; have written and/or reviewed Kubernetes APIs
+document, familiarity with project organization, conventions, and policies & procedures, as well as technical and/or writing abilities. Role-specific expectations, responsibilities, and requirements
+are enumerated below. Because github teams are not visible by nonmembers of the org and because changes
+to them are not transparent, the primary documentation of role membership beyond MEMBER should be 
+maintained in OWNERS files in the repository.
+
+- **MEMBER**:
+  - Requirements
+    - an active contributor for at least 3 months
+    - authored and/or reviewed at least 10 merged non-trivial PRs
```

Comment:
@cblecker Thanks for taking a look.

As for which repos this would apply to, that's explicitly TBD.

PRs and reviews: As written, equally, though I could imagine weighing reviews more heavily in order to encourage more reviewers.

Non-trivial: I wanted to allow discretion. There are some people who have submitted more than a dozen PRs, but all just fix typos using an automated mechanism. Those clearly shouldn't count.